src/lib/integrations/sit/fetchers.ts

```diff
@@ -12,8 +12,12 @@ function sitScheduleUrl(token: string, fromISO: string | null = null) {
     );
 }
 
+function sitGroupBookingUrl() {
```

Comment:
Any reason to not use a simple constant?

Reply:
Every other link was a function, so it was mostly for completeness. Can ofc change to a variable if you want?

Reply:
Yea, constant would be nice :upside_down_face: